src/components/patch/Patch.tsx

```diff
@@ -34,34 +35,35 @@ export class Patch extends React.Component<Props, State> {
       datetime: datetime,
       project: this.props.Patch.Version.identifier,
       author: this.props.Patch.Version.author,
+      builds: this.props.Patch.Builds
     };
   }
 
   public render() {
 
+    const Variants = () => (
+      <Grid container={true} spacing={24}>
+        <Grid item={true} xs={12}>
+          <Typography>
+            Applied at {this.state.datetime.format("D MMM YYYY, h:mm a")} on {this.state.project}
+          </Typography>
+        </Grid>
+        {Object.keys(this.state.builds).map(index => (
```

Comment:
Since this.state.builds is an array, what's the reason for enumerating its keys? It should return the arrays of the index. I think you can get the same behavior by doing this.state.builds.map and passing the object itself rather than the indices

Reply:
I have no particular reason for doing it this way besides a lack of familiarity with how the map function works. I tried just passing the object itself and you're right- it behaves the same.